pkg/cmd/stack/deploy.go

```diff
@@ -368,8 +369,16 @@ func deployServices(ctx context.Context, stack *model.Stack, k8sClient kubernete
 						return fmt.Errorf("service '%s' dependencies '%s' failed", svcName, strings.Join(failedJobs, ", "))
 					}
 					if failedServices := getServicesWithFailedProbes(ctx, stack, svcName, k8sClient); len(failedServices) > 0 {
-						for key, value := range failedServices {
-							return fmt.Errorf("service '%s' has failed his healthcheck probes: %s", key, value)
+						for service, err := range failedServices {
+							if oktetoErrors.IsLivenessProbeFailed(err) {
+								return fmt.Errorf("service '%s' has failed its healthcheck probes: %w", service, err)
+							} else if oktetoErrors.IsReadinessProbeFailed(err) {
+								if _, ok := serviceWarnings[service]; !ok {
+									message := fmt.Sprintf("service '%s' is failing its healthcheck probes: %s", service, err)
```

Comment:
Could we add the name of the service that is depending on this? I mean, something in the line of: `Service 'x' cannot be deployed because dependant service ' y'  is failing its healthcheck probes: ....`